src/index.ts

```diff
@@ -82,10 +79,7 @@ class HawkAPI {
     });
     this.app.use(express.json());
     this.app.use(bodyParser.urlencoded({ extended: false }));
-    this.app.use('/uploads', express.static(`./${process.env.UPLOADS_DIR || 'uploads'}`));
```

Comment:
Are you sure that we don't need this endpoint?

Reply:
yep, we use s3